protoc-options.txt

```diff
@@ -2,3 +2,4 @@
 --proto_path=./cosmos-sdk/proto
 --proto_path=./cosmos-sdk/third_party/proto
 --proto_path=./ibc-go/proto
+--proto_path=./third_party
```

Comment:
Do we need this to be in the `third_party` directory because it's imported that way? If not, could we move it to the root directory (`./google/protobuf`)?

I think this could be a submodule too: https://github.com/protocolbuffers/protobuf/tree/main/src/google/protobuf